compose/service.py

```diff
@@ -479,6 +488,7 @@ def build(self, no_cache=False):
             stream=True,
             rm=True,
             nocache=no_cache,
+            fileobj=self._get_fileobj(),
```

Comment:
This isn't quite right, fileobj would be for the whole context.  This needs to use the `dockerfile` kwarg.

https://github.com/docker/docker-py/blob/master/docker/client.py#L301

I don't think that option is in a release yet. I added it after the 1.0.0 release